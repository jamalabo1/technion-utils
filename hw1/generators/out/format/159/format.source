A X 585 1
C Z 760 2
C W 110 3
A Y 304 4
C W 881 5
B W 332 6
B Y 58 7
D W 799 8
D X 992 9
A W 799 10
C W 84 11
B X 478 12
A Z 139 13
A W 486 14
A Z 289 15
B X 403 16
B X 868 17
C Z 410 18
B Y 186 19
C Z 256 20
A Y 716 21
D Z 985 22
C X 485 23
D Y 704 24
D W 594 25
C X 55 26
C X 827 27
D Z 881 28
A Y 833 29
A W 188 30
C Z 86 31
A W 82 32
B Z 351 33
A W 600 34
D X 506 35
B W 399 36
B Z 222 37
D X 385 38
A W 514 39
B Z 55 40
C W 36 41
C Y 936 42
A W 147 43
A X 792 44
D W 321 45
D W 526 46
C Y 232 47
D W 654 48
C Y 552 49
D Y 563 50
A Y 638 51
B W 869 52
B Z 482 53
D X 418 54
D X 135 55
D X 740 56
B X 179 57
A X 356 58
B Z 18 59
C Y 767 60
C Z 715 61
A Z 596 62
B Z 682 63
B Z 638 64
B X 655 65
B X 772 66
C Z 53 67
D W 600 68