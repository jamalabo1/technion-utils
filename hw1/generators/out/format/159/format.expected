Blockchain Info:
1
Sender Name: D
Receiver Name: W
Transaction Value: 600
Transaction timestamp: 68
2
Sender Name: C
Receiver Name: Z
Transaction Value: 53
Transaction timestamp: 67
3
Sender Name: B
Receiver Name: X
Transaction Value: 772
Transaction timestamp: 66
4
Sender Name: B
Receiver Name: X
Transaction Value: 655
Transaction timestamp: 65
5
Sender Name: B
Receiver Name: Z
Transaction Value: 638
Transaction timestamp: 64
6
Sender Name: B
Receiver Name: Z
Transaction Value: 682
Transaction timestamp: 63
7
Sender Name: A
Receiver Name: Z
Transaction Value: 596
Transaction timestamp: 62
8
Sender Name: C
Receiver Name: Z
Transaction Value: 715
Transaction timestamp: 61
9
Sender Name: C
Receiver Name: Y
Transaction Value: 767
Transaction timestamp: 60
10
Sender Name: B
Receiver Name: Z
Transaction Value: 18
Transaction timestamp: 59
11
Sender Name: A
Receiver Name: X
Transaction Value: 356
Transaction timestamp: 58
12
Sender Name: B
Receiver Name: X
Transaction Value: 179
Transaction timestamp: 57
13
Sender Name: D
Receiver Name: X
Transaction Value: 740
Transaction timestamp: 56
14
Sender Name: D
Receiver Name: X
Transaction Value: 135
Transaction timestamp: 55
15
Sender Name: D
Receiver Name: X
Transaction Value: 418
Transaction timestamp: 54
16
Sender Name: B
Receiver Name: Z
Transaction Value: 482
Transaction timestamp: 53
17
Sender Name: B
Receiver Name: W
Transaction Value: 869
Transaction timestamp: 52
18
Sender Name: A
Receiver Name: Y
Transaction Value: 638
Transaction timestamp: 51
19
Sender Name: D
Receiver Name: Y
Transaction Value: 563
Transaction timestamp: 50
20
Sender Name: C
Receiver Name: Y
Transaction Value: 552
Transaction timestamp: 49
21
Sender Name: D
Receiver Name: W
Transaction Value: 654
Transaction timestamp: 48
22
Sender Name: C
Receiver Name: Y
Transaction Value: 232
Transaction timestamp: 47
23
Sender Name: D
Receiver Name: W
Transaction Value: 526
Transaction timestamp: 46
24
Sender Name: D
Receiver Name: W
Transaction Value: 321
Transaction timestamp: 45
25
Sender Name: A
Receiver Name: X
Transaction Value: 792
Transaction timestamp: 44
26
Sender Name: A
Receiver Name: W
Transaction Value: 147
Transaction timestamp: 43
27
Sender Name: C
Receiver Name: Y
Transaction Value: 936
Transaction timestamp: 42
28
Sender Name: C
Receiver Name: W
Transaction Value: 36
Transaction timestamp: 41
29
Sender Name: B
Receiver Name: Z
Transaction Value: 55
Transaction timestamp: 40
30
Sender Name: A
Receiver Name: W
Transaction Value: 514
Transaction timestamp: 39
31
Sender Name: D
Receiver Name: X
Transaction Value: 385
Transaction timestamp: 38
32
Sender Name: B
Receiver Name: Z
Transaction Value: 222
Transaction timestamp: 37
33
Sender Name: B
Receiver Name: W
Transaction Value: 399
Transaction timestamp: 36
34
Sender Name: D
Receiver Name: X
Transaction Value: 506
Transaction timestamp: 35
35
Sender Name: A
Receiver Name: W
Transaction Value: 600
Transaction timestamp: 34
36
Sender Name: B
Receiver Name: Z
Transaction Value: 351
Transaction timestamp: 33
37
Sender Name: A
Receiver Name: W
Transaction Value: 82
Transaction timestamp: 32
38
Sender Name: C
Receiver Name: Z
Transaction Value: 86
Transaction timestamp: 31
39
Sender Name: A
Receiver Name: W
Transaction Value: 188
Transaction timestamp: 30
40
Sender Name: A
Receiver Name: Y
Transaction Value: 833
Transaction timestamp: 29
41
Sender Name: D
Receiver Name: Z
Transaction Value: 881
Transaction timestamp: 28
42
Sender Name: C
Receiver Name: X
Transaction Value: 827
Transaction timestamp: 27
43
Sender Name: C
Receiver Name: X
Transaction Value: 55
Transaction timestamp: 26
44
Sender Name: D
Receiver Name: W
Transaction Value: 594
Transaction timestamp: 25
45
Sender Name: D
Receiver Name: Y
Transaction Value: 704
Transaction timestamp: 24
46
Sender Name: C
Receiver Name: X
Transaction Value: 485
Transaction timestamp: 23
47
Sender Name: D
Receiver Name: Z
Transaction Value: 985
Transaction timestamp: 22
48
Sender Name: A
Receiver Name: Y
Transaction Value: 716
Transaction timestamp: 21
49
Sender Name: C
Receiver Name: Z
Transaction Value: 256
Transaction timestamp: 20
50
Sender Name: B
Receiver Name: Y
Transaction Value: 186
Transaction timestamp: 19
51
Sender Name: C
Receiver Name: Z
Transaction Value: 410
Transaction timestamp: 18
52
Sender Name: B
Receiver Name: X
Transaction Value: 868
Transaction timestamp: 17
53
Sender Name: B
Receiver Name: X
Transaction Value: 403
Transaction timestamp: 16
54
Sender Name: A
Receiver Name: Z
Transaction Value: 289
Transaction timestamp: 15
55
Sender Name: A
Receiver Name: W
Transaction Value: 486
Transaction timestamp: 14
56
Sender Name: A
Receiver Name: Z
Transaction Value: 139
Transaction timestamp: 13
57
Sender Name: B
Receiver Name: X
Transaction Value: 478
Transaction timestamp: 12
58
Sender Name: C
Receiver Name: W
Transaction Value: 84
Transaction timestamp: 11
59
Sender Name: A
Receiver Name: W
Transaction Value: 799
Transaction timestamp: 10
60
Sender Name: D
Receiver Name: X
Transaction Value: 992
Transaction timestamp: 9
61
Sender Name: D
Receiver Name: W
Transaction Value: 799
Transaction timestamp: 8
62
Sender Name: B
Receiver Name: Y
Transaction Value: 58
Transaction timestamp: 7
63
Sender Name: B
Receiver Name: W
Transaction Value: 332
Transaction timestamp: 6
64
Sender Name: C
Receiver Name: W
Transaction Value: 881
Transaction timestamp: 5
65
Sender Name: A
Receiver Name: Y
Transaction Value: 304
Transaction timestamp: 4
66
Sender Name: C
Receiver Name: W
Transaction Value: 110
Transaction timestamp: 3
67
Sender Name: C
Receiver Name: Z
Transaction Value: 760
Transaction timestamp: 2
68
Sender Name: A
Receiver Name: X
Transaction Value: 585
Transaction timestamp: 1